.changeset/shiny-regions-teach.md

```
@@ -1,5 +1,5 @@
 ---
-"task-master-ai": major
+"task-master-ai": minor
 ---
 
 @tm/cli: add auto-update functionality to every command
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)